manifests/bootstrap_rpm.pp

```diff
@@ -0,0 +1,61 @@
+# This class builds an RPM containing the bootstrap for a subscription-manager consumer
+# This file is placed in $katello_www_pub_dir.
+# @api private
+class foreman_proxy_content::bootstrap_rpm (
+  $hostname                          = $facts['networking']['fqdn'],
+  $deployment_url                    = '/rhsm',
+  $rhsm_port                         = 443,
+  $rhsm_ca_dir                       = '/etc/rhsm/ca',
+  $candlepin_cert_rpm_alias_filename = 'katello-ca-consumer-latest.noarch.rpm',
+  $katello_www_pub_dir               = '/var/www/html/pub',
+  $katello_server_ca_cert            = $certs::katello_server_ca_cert,
+  $server_ca_name                    = $certs::server_ca_name,
+  $ca_cert                           = $certs::ca_cert,
+  $server_ca                         = $certs::server_ca,
+) {
+  include certs
```

Comment:
This doesn't really work. The class parameters are compiled and determined before the body so at this point `$katello_server_ca_cert` may be empty.